samples/drivers/adc/README.rst

```diff
@@ -0,0 +1,56 @@
+.. _adc-sample:
+
+Analog-to-Digital Converter (ADC)
+#################################
+
+Overview
+********
+
+This sample demonstrates how to use the ADC driver API.
+
+Depending on the MCU type, it reads the ADC samples of one or two ADC channels
+and prints the raw readings to the console.
+
+The pins of the ADC channels are board-specific. Please refer to the board
+or MCU datasheet for further details.
+
+The sample should work with most MCU series, including STM32, NXP Kinetis and
+Microchip SAM0. Nordic nRFx MCUs are currently not supported, as they require
+additional configuration for their differential input feature.
```

Comment:
```suggestion
Microchip SAM0. Nordic nRFx MCUs are currently not supported.
```

The stated  reason why is not totally accurate, so better to just say they're not supported IMO.